src/ui/control-panel.ts

```diff
@@ -610,7 +610,9 @@ class ControlPanel extends Container {
         const scaleInput = new NumericInput({
             class: 'control-element-expand',
             precision: 2,
-            value: 1
+            value: 1,
+            min: 0.01,
```

Comment:
if they can scale up by 10k, why not down as well?

Reply:
I tried, but `min` must be a multiple of `precision` (which itself must be 2 for natural scaling).